lib/challenge.js

```diff
@@ -33,7 +42,10 @@
 var pet_owner;
 
 // Answer Starts Here
-
+pet_owner = {
+  name : "Saphron Reyn",
+  address : "1 Second St, Cambridge, MA, 02141",
+}
```

Comment:
Your object literal is good in that it will run correctly. Run `grunt nag` if you'd like to further polish it; there are some ways it could be better.

1